osu.Game/Rulesets/Edit/HitObjectComposer.cs

```diff
@@ -212,6 +217,8 @@ private void showGridFor(IEnumerable<HitObject> selectedHitObjects)
             {
                 distanceSnapGridContainer.Child = distanceSnapGrid;
                 distanceSnapGridContainer.Show();
+
+                lastGridUpdateTime = EditorClock.CurrentTime;
```

Comment:
can't this just be moved outside of the `if` to avoid duplicating it in two places?

Reply:
The other one is in a completely separate block, I wanted to avoid potentially calling a secondary `showGridFor()` by adding it here too. With that being said, it should be moved out of the `if` block here.

Reply:
Misunderstood, this method is being called.